safesocket/safesocket_darwin.go

```diff
@@ -49,6 +50,8 @@ func localTCPPortAndTokenMacsys() (port int, token string, err error) {
 	return port, auth, nil
 }
 
+var warnAboutRootOnce sync.Once
```

Comment:
Add a comment why here.

Reply:
Added some comments where it's actually used, but this one seemed self-explanatory.